dbfit-java/core/src/main/java/dbfit/api/vendor/DefaultDbStoredProcedureCall.java

```diff
@@ -1,25 +1,29 @@
-package dbfit.api;
+package dbfit.api.vendor;
 
+import dbfit.api.DBEnvironment;
 import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.DbParameterAccessors;
-import static dbfit.util.sql.PreparedStatements.buildFunctionCall;
-import static dbfit.util.sql.PreparedStatements.buildStoredProcedureCall;
+import static dbfit.util.sql.PreparedStatements.buildStoredRoutineCallCmdText;
 
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 
-public class DbStoredProcedureCall {
+public class DefaultDbStoredProcedureCall implements DbStoredProcedureCall {
     private final DBEnvironment environment;
     private final String name;
     private final DbParameterAccessors accessors;
 
-    public DbStoredProcedureCall(DBEnvironment environment, String name, DbParameterAccessor[] accessors) {
+    public DefaultDbStoredProcedureCall(DBEnvironment environment, String name, DbParameterAccessor[] accessors) {
         this.environment = environment;
         this.name = name;
         this.accessors = new DbParameterAccessors(accessors);
     }
 
+    /* (non-Javadoc)
+     * @see dbfit.api.IDbStoredProcedureCall#getName()
```

Comment:
That's an _oops_ created by Eclipse refactoring functionality. Apologies for the sloppyness.